tests/tests_unit/test_data_classes/test_data_models/test_data_types.py

```diff
@@ -53,7 +69,7 @@ def test_load_ignore_unknown_properties(self) -> None:
     def test_load_dump_unkown_property(self) -> None:
         data = {"type": "unknowngibberish", "list": True, "unit": {"externalId": "known"}}
         obj = PropertyType.load(data)
-        assert isinstance(obj, UnknownPropertyType)
+        assert isinstance(obj, UnknownCogniteObject)
```

Comment:
Changed this one as `UnknownPropertyType` will likely be removed in next major version.